### ISSUE TITLE
How to just the package of utp ? 

### ISSUE THREAD

**Issue by ggandycong** [bittorrent/libutp#120]
In default, the max package of utp is 1444 size(about 1k),capturing packets with **WireShark** on linux(as follow).
![image](https://user-images.githubusercontent.com/17496760/62033458-8fa18100-b21e-11e9-8aeb-730067f314f8.png)

However, I want to enlarge the size of package to **16k** ,or **64k**. I try to do it, but it have no changes.
```
#define REORDER_BUFFER_MAX_SIZE 1024* 10
#define OUTGOING_BUFFER_MAX_SIZE 1024 * 10
#define PACKET_SIZE 1435 * 10
```
So, How to do it? It can send greater package ?

**Comment by lighterowl** [comment]
You do realise that this will get fragmented anyway due to the network interface's MTU, right?

**Comment by ggandycong** [comment]
Yes,I know.  but I think that it can reduce the amount of package when send in a big file.  the size of package is more than the limit of MTU, it will be split when send  and   merge it when receive.
